ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.js

```diff
@@ -267,6 +267,28 @@ export default {
         },
       },
     },
+    {
+      id: 'UWU 爆岩',
+      netRegex: NetRegexes.startsUsing({ id: '2B6A', source: '爆破岩石', capture: false }),
+      suppressSeconds: 1,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '爆岩',
+        },
+      },
+    },
+    {
+      id: 'UWU 流沙',
+      netRegex: NetRegexes.startsUsing({ id: '2B65', source: '泰坦', capture: false }),
+      suppressSeconds: 1,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          cn: '流沙',
```

Comment:
On this one, it looks like this is Weight of the Land, the ground puddles? As a general rule, we usually don't have triggers set up for visible ground effects like this. That being said, I haven't run Ultima myself, so I can't really comment. @quisquous would know better on this one.

Reply:
It maybe not visible for MT when he is not at the same side with crowd.

Reply:
or you can add a condition after netRegex(es):
`condition: (data) => data.role === 'tank',`

Reply:
Sorry, I think I agree with @JLGarber on this one.  I think in general I'd prefer no triggers for ground effects because it gets very noisy.

This particular trigger is maybe better for your user folder.

Reply:
@quisquous I have a question about this. There is another groud effect ability(the laser when UWU's HP is lower than 50%, I don't know it's name in English) in UWU, and it's random, chose 1 from 3 choies(which can be distinguished by different ability id). It's not that visible like Land Weight and happened in the same time with another groud effect. Should I issue a pr for this?


![image](https://user-images.githubusercontent.com/13553903/104355807-27477c00-5546-11eb-9fd6-497b58de1f8a.png)